AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -300,15 +300,18 @@ bloody-disgusting.com,haberolduk.com,askdifference.com,multics.eu,pipeflare.io,t
 !#endif
 !
 ! NOTE: 'mdpDeBlocker'
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=noopjs,domain=texture-packs.com|xdarom.com
+||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=noopjs,domain=texture-packs.com|xdarom.com|katoikos.world
```

Comment:
Better use scriptlet - redirects don't help if used DNS filtering (with apps).

Also this rule does not work on my end.

Reply:
@Alex-302 yeah, the 2 scriptlet rule that I included seems enough, I remove the redirect